Makefile

```diff
@@ -214,8 +214,17 @@ dev/syncdeps:
 
 .PHONY: release/versionbump
 release/versionbump:
+	bazel run //hack/versionbump:versionbump -- patch $(VERSION) > $(PWD)/version.txt
+	# Reload make so it rereads version.txt
+	$(MAKE) release/gen-files
+
+.PHONY: release/gen-templates
```

Comment:
Please add documentation to all of these make commands so that I understand what is happening here.   Also, do we have a test that validates that the produced files are valid YAML?  How can we test the outputs better?

Reply:
Also, we now edit the templates and not the actual files?  Can we add a comment in the actual file to where the template lives?

Reply:
- Added comments to `Makefile`
- Added YAML validation. It just loads the generated file in order to verify if it's a valid YAML file. We can add strict schemas in the future if we want.
- Added the source template location in the warning.